vendor/k8s.io/kubernetes/pkg/kubelet/kubelet_server_journal.go

```diff
@@ -137,21 +146,26 @@ func (a *journalArgs) Copy(w io.Writer) {
 	// set the deadline to the maximum across both runs
 	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(time.Duration(a.Timeout)*time.Second))
 	defer cancel()
-	// show the previous boot if possible, eating errors
-	a.copyForBoot(ctx, w, 1)
-	// show the current boot
-	a.copyForBoot(ctx, w, 0)
+	if len(a.Boot) > 0 {
+		a.copy(ctx, w)
+	} else {
+		// show the previous boot if possible, eating errors
+		a.Boot = "-1"
```

Comment:
I do not like this flow - please go back to providing a boot identifier as an argument (which must be int, not string), and you need to pass a boolean to suppress the error message.